eng/common/TestResources/build-test-resource-config.yml

```diff
@@ -14,6 +14,24 @@ parameters:
     default: null
 
 steps:
+  - task: AzurePowerShell@5
+    displayName: Set Pipeline Subnet Info
+    env: ${{ parameters.EnvVars }}
+    inputs:
+      azureSubscription: Azure SDK Engineering System
```

Comment:
Weren't you planning to use the azure-sdk-tests identity for this?

Reply:
Ah yes, thanks for the reminder, reverted and testing.